camelot/view/action_steps/application.py

```diff
@@ -62,7 +62,9 @@ def __init__(self, admin):
     def render( self, gui_context ):
         """create the main window. this method is used to unit test
         the action step."""
-        from ..mainwindow import MainWindow
+        from ..mainwindowproxy import MainWindowProxy
+        from camelot.view.register import register
```

Comment:
ik zou die proxy niet in de register steken, dat is niet nodig, want die heeft een parent,
enkel de mainwindow die zelf geconstrueerd w, moet in de register gestoken w

Reply:
done